### PR TITLE
S2U-42 SAK-49261 Card Game: Instructors with full access can play with groups

### PR DESCRIPTION
**Pull request by stetsche**
This addresses [SAK-49261](https://sakaiproject.atlassian.net/browse/SAK-49261) for [S2U-42](https://sakaiproject.atlassian.net/browse/S2U-42)

[SAK-49261]: https://sakaiproject.atlassian.net/browse/SAK-49261?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[S2U-42]: https://sakaiproject.atlassian.net/browse/S2U-42?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by stetsche** [comment]
Here some screenshots as context for the i18n strings:
i18n_group
i18n_select_scope
i18n_site

File: roster2/bundle/src/bundle/card_game_es.properties [hunk 1]
```diff
@@ -21,3 +21,10 @@ whats_the_user_name = ¿Cu\u00E1l es el nombre del estudiante?
 user_progress = {0} de {1} intentos fueron correctos para este estudiante.
 mark_learned = Conozco a este estudiante, no me lo muestres en esta ronda.
 no_name_found = No se ha encontrado ning\u00fan nombre para esta entrada.
+group_scope_display = del grupo {0}
+group_scope_radio = Nombres de los miembros de un grupo
+groups_label = Grupo para aprender:
+scope_radio_label = ¿Qu\u00e9 nombres quieres aprender?
+select_scope = Empezar a aprender
+site_scope_display = de todo el sitio
+site_scope_radio = Nombres de todos los participantes del sitio
```
Comment (bgarciaentornos) [inline roster2/bundle/src/bundle/card_game_es.properties:30]:
```suggestion
site_scope_radio = Nombres de todos los participantes del sitio

```

File: roster2/tool/src/webapp/js/card-game/card-game.js [hunk 2]
```diff
@@ -344,8 +427,32 @@ export default class CardGame extends BaseGame {
...
+                        .map(userId => this.state.allUsers.find(user => user.id === userId))
+                        .filter(user => typeof user !== "undefined");
+        } else {
+            return [];
+        }
+    }
+
     calcLearnUsers() {
-        return this.state.allUsers.filter((user) => !isUserLearned(user, this.config));
+        console.log("learnable", this.state.learnableUsers)
```
Comment (bgarciaentornos) [inline roster2/tool/src/webapp/js/card-game/card-game.js:454]: Is this log needed?

File: roster2/tool/src/webapp/js/card-game/card-game.js [hunk 3]
```diff
@@ -357,15 +464,16 @@ export default class CardGame extends BaseGame {
     }
 
     calcUserOptionsData() {
+        console.log(this.state.learnUsers)
```
Comment (bgarciaentornos) [inline roster2/tool/src/webapp/js/card-game/card-game.js:467]: Is this log needed?

Comment (stetsche) [reply roster2/tool/src/webapp/js/card-game/card-game.js]: nope, removed.

File: roster2/tool/src/webapp/js/card-game/card-game.js (hunk 2 above)
Comment (stetsche) [reply roster2/tool/src/webapp/js/card-game/card-game.js]: nope, removed.

**Review by bgarciaentornos — APPROVED** [review]

**Review by mpellicer — APPROVED** [review]
Thanks for the review @bgarciaentornos

**Pull request merged**